<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1873</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbltitulo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registro de usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingrese su nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellido1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingrese primer apellido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellido2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>250</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingrese segundo apellido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDNI">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>160</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingrese DNI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>601</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingrese Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Ingrese Contraseña</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbRol">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Cliente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recepcionista</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mecánico</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>680</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtConfirmPassword">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>460</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirme contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>680</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QWidget" name="imagen" native="true">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>50</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1873</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
